### PR TITLE
only sync mac_arm for 4.12+

### PR DESCRIPTION
**Pull request by Ximinhan**
wait to confirm from https://redhat-internal.slack.com/archives/CB95J6R4N/p1681459312377089
update: currently mac_arm64 only support 4.12+

### CODE REVIEW

File: pyartcd/pyartcd/pipelines/operator_sdk_sync.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def _extract_binaries(self, arch, sdkVersion, build):
         self.exec_cmd(cmd)
         if arch == 'amd64' or arch == 'arm64':
             tarballFilename = f"{self.sdk}-{sdkVersion}-darwin-{rarch}.tar.gz"
-            share_path = "mac_arm64" if arch == 'arm64' else "mac"
+            share_path = "mac_arm64" if arch == 'arm64' and int(self.assembly.split('.')[1])>=12 else "mac"
```
Comment (vfreex) [inline pyartcd/pyartcd/pipelines/operator_sdk_sync.py:86]: Please use `util.isolate_major_minor_in_group` to determine the group version and explicitly check if it is greater than or equal to 4.12.

Comment (Ximinhan) [reply pyartcd/pyartcd/pipelines/operator_sdk_sync.py]: updated

File: pyartcd/pyartcd/pipelines/operator_sdk_sync.py [hunk 2]
```diff
@@ -83,7 +83,8 @@ def _extract_binaries(self, arch, sdkVersion, build):
         self.exec_cmd(cmd)
         if arch == 'amd64' or arch == 'arm64':
             tarballFilename = f"{self.sdk}-{sdkVersion}-darwin-{rarch}.tar.gz"
-            share_path = "mac_arm64" if arch == 'arm64' else "mac"
+            _, minor = util.isolate_major_minor_in_group(self.group)
+            share_path = "mac_arm64" if arch == 'arm64' and minor >= 12 else "mac"
```
Comment (joepvd) [inline pyartcd/pyartcd/pipelines/operator_sdk_sync.py:87]: What happens if we have openshift5?

Comment (Ximinhan) [reply pyartcd/pyartcd/pipelines/operator_sdk_sync.py]: updated

**Review by joepvd — APPROVED** [review]
/lgtm

**Pull request merged**